app/mailers/curator_mailer.rb

```diff
@@ -2,12 +2,12 @@ class CuratorMailer < ApplicationMailer
 
   def released_records(users, records)
 #    @records = records
-#    mail(from: ENV['mailer_sender'], to: users.map(&:email), subject: 'Metadata Curation Tool Released Records')
```

Comment:
I checked, this value is not in igit or in the bamboo variables list, so I don't think there is anything that needs to be removed outside of this code. The only reference I could find was in application.yml.